{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/tests/test_views.py

```diff
@@ -91,6 +92,13 @@ def test_authenticated(self, user: User, rf: RequestFactory):
 
         assert response.status_code == 200
 
+    def test_authenticated_integration(self, user: User, client: Client):
+        client.force_login(UserFactory())
+        response = client.get(
+            reverse("users:detail", kwargs={"username": user.username})
+        )
+        assert response.status_code == 200
```

Comment:
I think it would be nice to include some tests covering this, but I'm not happy where it is right now... 

I'm thinking that it might make more sense to be in a new file called `test_integration.py` (or something like this) to signal that it's for testing the whole request/response, from URL, view, form all the way to the DB, and that these are expected to be slower. We could even add a pytest marker for them.

That starting to sound like a bigger change, and I might save it for another PR. Thoughts?

Reply:
Is it an idea to move it to utils and call it `test_webpack_loader`?